setup.py

```diff
@@ -102,7 +102,7 @@ def check_setuptools_features():
         'logstats==0.2.1',
         'base58==0.2.2',
         'flask==0.10.1',
-        'requests==2.9',
+        'requests>=2.9',
```

Comment:
I propose we still use `2.9`, otherwise, if a lower version of `requests` is already installed, a more recent version will not get install unless the option `--upgrade` is used.

```bash
$ pip install requests==2.0
Collecting requests==2.0
  Downloading requests-2.0.0-py2.py3-none-any.whl (391kB)
    100% |████████████████████████████████| 399kB 2.3MB/s 
Installing collected packages: requests
Successfully installed requests-2.0.0
$ pip install requests~=2.0
Requirement already satisfied (use --upgrade to upgrade): requests~=2.0 in /usr/local/lib/python3.5/site-packages
$ pip install --no-cache-dir requests~=2.0
Requirement already satisfied (use --upgrade to upgrade): requests~=2.0 in /usr/local/lib/python3.5/site-packages
$ pip install --upgrade requests~=2.0
Collecting requests~=2.0
  Downloading requests-2.11.1-py2.py3-none-any.whl (514kB)
    100% |████████████████████████████████| 522kB 1.6MB/s 
Installing collected packages: requests
  Found existing installation: requests 2.0.0
    Uninstalling requests-2.0.0:
      Successfully uninstalled requests-2.0.0
Successfully installed requests-2.11.1
```